spec/csv_generator/pacbio_sample_sheet_spec.rb

```diff
@@ -331,7 +333,7 @@
           create(:pacbio_well, generate_hifi: 'On Instrument', ccs_analysis_output: 'No',
                                pools: pool2)
         end
-        let(:plate) { create(:pacbio_plate, wells: [well1, well2]) }
```

Comment:
I can't see any references to plate (all instances). Maybe you can just remove it entirely. I think its to ensure the wells have the same plate but the tests aren't checking it.

Reply:
Hm what do you mean? I refactored it a little so the wells are defined in either test with different pools etc, then used at the very top of the file to create the plate>run>csv.